### ISSUE TITLE
Show usage data

### ISSUE THREAD

**Issue by cassidyjames** [elementary/appcenter#597]
It could be interesting to show usage data, kind of like Steam's "time played" on the app listing. This could be combined with the total amount paid to communicate value.

This of course assumes we can get the usage data (e.g. from Zeitgeist?), and locally store the amount paid.

Thanks to @ted-gould via [Mastodon](https://social.gould.cx/@ted/99474736508309757) for the idea. :wink:

**Comment by peteruithoven** [comment]
This would be user specific, only stored locally?

**Comment by ted-gould** [comment]
On U8 that is how we implemented application usage tracking. The data was Open/Close events on the applications stored in the Zeitgeist database. The ZG database was never taken off of the device.

**Comment by cassidyjames** [comment]
@peteruithoven yes, local and per-user. If at some point we introduced an account system, this would likely be synced as well, but that's out of scope for this issue.
